examples/routing_service/routing_service_with_security/README.md

```diff
@@ -0,0 +1,75 @@
+# Routing Service with Security example
+
+## Concept
+Routing Service supports DDS security, and may be configured to bridge 
+topic data between:
+- 2 unsecured domains (no DDS security) or 
+- 1 secured domain and 1 unsecured domain or
+- 2 secured domains
+
+## Example description
+This example demonstrates Routing Service bridging 
+1 secured domain and 1 unsecured domain using the RTI DDS Ping utility.
+The rtiddsping publisher resides in Domain 0, an unsecured domain and 
+the rtiddsping subscriber resides in Domain 1, a secured domain.
+
+---------------------------          ---------------------------
+|   Domain 0 (unsecured)  |          |   Domain 1 (secured)    |
+|                         |          |                         |
+|  |--------------|    |----------------|    |--------------|  |
+|  |  rtiddsping  |--->| Routing Service|--->|  rtiddsping  |  |
+|  |   Publisher  |    |                |    |  Subscriber  |  |
+|  |--------------|    |----------------|    |--------------|  |
+|                         |          |                         |
+---------------------------          ---------------------------
+
+In the secured domain, security artifacts need to be created for all 
+communicating participants,
+i.e. the rtiddsping participant and the Routing Service participant.
+The security artifacts in this example are located in 
+rticonnextdds-examples/resources/certAuthority. 
+The certificates for this example are signed based on the DSA algorithm.
+
+In this example, data is only routed from Domain 0 to Domain 1.
+
+
+## Before running the example
+Linux-based
+-----------
+If using dynamic libraries, your LD_LIBRARY_PATH must include
+$NDDSHOME/lib/<architecture> and 
```

Comment:
You may want to use "code" (grave accent `) for paths and variables. They will look better this way. 

Example: `$NDDSHOME/lib/<architecture>`

See [the Markdown help](https://guides.github.com/features/mastering-markdown/) for further info.

Reply:
Done.